src/mondemand_vmstats.erl

```diff
@@ -382,10 +388,51 @@ scheduler_wall_time_diff (PrevSchedWallTime, SchedWallTime) ->
     <- lists:zip(lists:sort(PrevSchedWallTime),lists:sort(SchedWallTime))
   ].
 
+% Sum of current carriersizes(mcbs+scbs) for all instances although
+% currently there is only one global instance for literal_alloc.
+% Note: This function returns constant value zero when OTP version
+% is 18 or before.
+-ifdef (namespaced_types_19_later).
```

Comment:
I don't like 'namespaced_types_19_later' as that's not what this is.  I believe it's actually something like 'allocator_stats_available', right?  This doesn't have anything to do with types AFAIK.

Reply:
@djnym Thank you, updated.